### PR TITLE
Add link to api settings

### PR DESCRIPTION
**Pull request by kandros**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ npm install habitica --save
 
 ## Usage
 
-This package is intentionally light and unopinionated. It should be used in conjunction with [Habitica's API documentation](https://habitica.com/apidoc/).
-
+This package is intentionally light and unopinionated. It should be used in conjunction with [Habitica's API documentation](https://habitica.com/apidoc/).  
+You get find your user id and api token [here](https://habitica.com/#/options/settings/apis)
```
Comment (crookedneighbor) [inline README.md]: The link is `https://habitica.com/#/options/settings/api`

File: README.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ npm install habitica --save
 
 ## Usage
 
-This package is intentionally light and unopinionated. It should be used in conjunction with [Habitica's API documentation](https://habitica.com/apidoc/).
-
+This package is intentionally light and unopinionated. It should be used in conjunction with [Habitica's API documentation](https://habitica.com/apidoc/).
```
Comment (crookedneighbor) [inline README.md]: Remove the extra space here.

**Review by crookedneighbor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kandros) [reply README.md]: ups, fixing

File: README.md [hunk 3]
```diff
@@ -14,6 +14,7 @@ npm install habitica --save
 
 This package is intentionally light and unopinionated. It should be used in conjunction with [Habitica's API documentation](https://habitica.com/apidoc/).
 
+You get find your user id and api token [here](https://habitica.com/#/options/settings/api).
```
Comment (crookedneighbor) [inline README.md]: "You get find" -> "You can find"

**Review by crookedneighbor — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (kandros) [reply README.md]: today feels like

**Comment by crookedneighbor** [comment]
:+1:

**Pull request merged**